### PR TITLE
ADD: implete skin_classifier with Cython to speedup

### PR DESCRIPTION
**Pull request by goaxe**

### CODE REVIEW

**Comment by deniszh** [comment]
:+1:

**Comment by Tatsh** [comment]
We really need this. It is very slow for large images.

**Comment by Tatsh** [comment]
This is very good, almost a 5x increase in speed.

**Comment by hhatto** [comment]
Thanks

**Comment by Tatsh** [comment]
Couple of issues. I am noting them in the review. You should not release a new version until you fix them.

File: setup.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
+from Cython.Build import cythonize
```
Comment (Tatsh) [inline setup.py]: This should be in a `try/except ImportError`. On error, tell the user to install cython (`pip install cython`) as it is required for the `ext_modules` line. And, perhaps on line 27 'Cython' can be removed.

File: nude.py [hunk 2]
```diff
@@ -19,15 +18,8 @@ def is_nude(path_or_io):
 
 class Nude(object):
 
-    Skin = namedtuple("Skin", "id skin region x y checked")
-
     def __init__(self, path_or_io):
-        if isinstance(path_or_io, Image.Image):
```
Comment (Tatsh) [inline nude.py]: The removal of this code may be a breaking change, unless `Image.open()` can accept an `Image.Image` instance too (perhaps it just returns it back).

**Comment by hhatto** [comment]
How does that look? #18

**Pull request merged**